aligned_accessor/aligned_accessor.md

```diff
@@ -0,0 +1,294 @@
+  return vectorized_norm(y);
+}
+```
+
+# Wording
+
+> Text in blockquotes is not proposed wording, but rather instructions for generating proposed wording.
+> The � character is used to denote a placeholder section number which the editor shall determine.
+>
+> In *[version.syn]*, add
+
+```c++
+#define __cpp_lib_aligned_accessor YYYYMML // also in <mdspan>
+```
+
+> Adjust the placeholder value as needed so as to denote this proposal's date of adoption.
+>
+> To the Header `<mdspan>` synopsis **[mdspan.syn]**, after `class default_accessor` and before `class mdspan`, add the following.
+
+```c++
+// [mdspan.accessor.aligned], class template aligned_accessor
+template<class ElementType, size_t byte_alignment>
+  class aligned_accessor;
+```
+
+> At the end of **[mdspan.accessor.default]** and before **[mdspan.mdspan]**, add all the material that follows.
+
+## Add subsection � [mdspan.accessor.aligned] with the following
+
+<b> � Class template `aligned_accessor` [mdspan.accessor.aligned] </b>
+
+<b> �.1 Overview [mdspan.accessor.aligned.overview] </b>
+
+```c++
+template<class ElementType, size_t the_byte_alignment>
+struct aligned_accessor {
+  using offset_policy = default_accessor<ElementType>;
+  using element_type = ElementType;
+  using reference = ElementType&;
+  using data_handle_type = ElementType*;
+
+  static constexpr size_t byte_alignment = the_byte_alignment;
+
+  constexpr aligned_accessor() noexcept = default;
+
+  template<class OtherElementType, size_t other_byte_alignment>
+    constexpr aligned_accessor(
+      aligned_accessor<OtherElementType, other_byte_alignment>) noexcept;
+
+  constexpr operator default_accessor<element_type>() const {
+    return {};
+  }
+
+  constexpr reference access(data_handle_type p, size_t i) const noexcept;
+
+  constexpr typename offset_policy::data_handle_type
+    offset(data_handle_type p, size_t i) const noexcept;
+
+  constexpr static bool is_sufficiently_aligned(data_handle_type p);
+};
+```
+
+[1]{.pnum} *Mandates*: `byte_alignment` is a power of two.
+
+[2]{.pnum} `aligned_accessor` meets the accessor policy requirements.
+
+[3]{.pnum} `ElementType`  is required to be a complete object type that is neither an abstract class type nor an array type.
+
+[4]{.pnum} Each specialization of `aligned_accessor` is a trivially copyable type that models `semiregular`.
+
+[5]{.pnum} $[0, n)$ is an accessible range for an object `p` of type `data_handle_type` and an object of type `aligned_accessor` if and only if $[$`p`, `p` + $n)$ is a valid range.
+
+## Members [mdspan.accessor.aligned.members]
+
+```c++
+template<class OtherElementType, size_t other_byte_alignment>
+  constexpr aligned_accessor(
+    aligned_accessor<OtherElementType, other_byte_alignment>) noexcept;
+```
+
+[1]{.pnum} *Constraints*:
+
+* [1.1]{.pnum} `is_convertible_v<OtherElementType(*)[], element_type(*)[]>` is `true`, and
+
+* [1.2]{.pnum} `gcd(other_byte_alignment, byte_alignment) == byte_alignment` is `true`.
+
```

Comment:
Conversion operator to default_accessor is also missing

Reply:
It's defined inline in the class overview, so we shouldn't need wording for it.